### DISCUSSION TITLE
Bluetooth earphones?

### DISCUSSION THREAD

**Discussion by slowglow** [cloudedbats/cloudedbats_wurb_2020#44] [Ideas]
Could be interesting to have the feedback coming over bluetooth earphones. That way the user can be significant distance away from the device and still operate it remotely AND have the audio feedback. 

Could this also be of benefit for potentially avoiding problems with the stability of the simultaneous audio recording and earphone feedback? I remember it was a struggle to achieve stability with the pyaudiodevice library, which was eventually replaced.   

I have no idea how difficult it would be to achieve it though.

**Comment by arnoldandreasson** [comment]
Yes, it should be useful and I have tried to find simple solutions for that, but it is not that easy when running a "headless" Raspberry Pi. Maybe in the future, and there is always a possibility to buy a 3.5 mm to bluetooth adapter and use that right now. I don't think the detector code needs to be modified, it is all about configuring the linux system.

Personally I'm more interested in streaming sound over internet. Then you can use the same client hardware that is used to control the detector, and is it a station connected to internet then it is much more useful that bluetooth.
But those are extra features and there is a huge list of basic functionality that is much higher on the prio list.

Good to have it here on the discussion page anyway if someone else know more about a "simple enough" solution. (RaspAP was a similar really handy solution for another problem that was too complicated to do from the terminal window.)
